### PR TITLE
Worm Bundles!

### PR DESCRIPTION
**Pull request by Stagrawl**
https://github.com/Blackstone-SS13/BLACKSTONE/assets/145404209/52f5f5e4-6ffb-49dd-8099-8ef2b73650fd

Allows you to bundle worms. This is supposed to be a feature, I think, just never got added properly.

### CODE REVIEW

File: code/game/objects/items/rogueitems/natural/clothfibersthorn.dm [hunk 1]
```diff
@@ -342,3 +342,36 @@
...
+			if(wormcount == 1)
+				new /obj/item/natural/worms(get_turf(user))
+					wormcount--
+			else if(wormcount >= 2)
+				var/obj/item/natural/bundle/worms/B = new(get_turf(user))
+				B.amount = clamp(wormcount, 2, 12)
+				B.update_bundle()
+				wormcount -= clamp(wormcount, 2, 12)
+		for(var/obj/item/natural/worms/F in get_turf(src))
+			qdel(F)
```
Comment (JTGSZ) [inline code/game/objects/items/rogueitems/natural/clothfibersthorn.dm:375]: this is gonna delete your single worm produced in a condition above you made if you do get it.

Comment (JTGSZ) [reply code/game/objects/items/rogueitems/natural/clothfibersthorn.dm:375]: wait nvm i didn't look at the get_turf

File: code/game/objects/items/rogueitems/natural/clothfibersthorn.dm [hunk 2]
```diff
@@ -342,3 +342,36 @@
...
+		while(wormcount > 0)
+			if(wormcount == 1)
+				new /obj/item/natural/worms(get_turf(user))
+					wormcount--
+			else if(wormcount >= 2)
+				var/obj/item/natural/bundle/worms/B = new(get_turf(user))
+				B.amount = clamp(wormcount, 2, 12)
+				B.update_bundle()
+				wormcount -= clamp(wormcount, 2, 12)
+		for(var/obj/item/natural/worms/F in get_turf(src))
```
Comment (JTGSZ) [inline code/game/objects/items/rogueitems/natural/clothfibersthorn.dm:374]: actually yeah, what happens if the user is standing ontop of the worms, he loses one.

Comment (Stagrawl) [reply code/game/objects/items/rogueitems/natural/clothfibersthorn.dm:374]: Oh, that's fucked. I never noticed that in testing.

Comment (JTGSZ) [reply code/game/objects/items/rogueitems/natural/clothfibersthorn.dm:374]: I guess either give it a escape message, or make it not lose the singular worm as idk if this is intended on your part.

**Comment by lovegreenstuff** [comment]
as i copypasted my code, so does my code get copypasted
the serpent consumes its own tail once more

**Comment by Stagrawl** [comment]
Someone tried to implement worm stacking previously but never carried through. I'll see if I can fix the 3rd sprite thing.

The reason the worm gets deleted is so you're not left with a spare worm + the bundle containing two worms.

https://github.com/Blackstone-SS13/BLACKSTONE/assets/145404209/63a79b5b-0231-47df-ba9e-238543231935

**Comment by JTGSZ** [comment]
this bundling code is confusing me greatly, but it is a copy-paste of everything else soooo

**Pull request merged**